packages/myst-execute/tests/run.spec.ts

```diff
@@ -78,6 +78,7 @@ casesList.forEach(({ title, cases }) => {
         file.path = path.join(__dirname, 'notebook.ipynb');
 
         await kernelExecutionTransform(before, file, {
+	      basePath: __dirname,
           sessionFactory: async () => await sessionManagerFactory.load(),
```

Comment:
I guess linting isn't on in the `*.spec.ts` files. GIthub puts 8 spaces on tabs... looks strange. :)
```suggestion
          basePath: __dirname,
          sessionFactory: async () => await sessionManagerFactory.load(),
```